cmd/check.go

```diff
@@ -1,20 +1,67 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	"github.com/cuducos/minha-receita/check"
 )
 
+var (
+	checksumTargetDir string
+	deleteZipFiles    bool
+)
+
+const checksumHelper = `
+Checksum of the downloaded files.
+
+Even though the official website of the Brazilian Federal Revenue does not offer 
+a checksum for their files, this command can be used to create or check the checksum
+of downloaded files.`
+
+var rootChecksumCmd = &cobra.Command{
+	Use:   "checksum",
+	Short: "Checksum of the downloaded files.",
+	Long:  checksumHelper,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return fmt.Errorf("error, you must also specify a checksum command, like create or check")
+	},
+}
+
+const createChecksumHelper = "Creates checksum of downloaded files."
+
+var createChecksumCmd = &cobra.Command{
+	Use:   "create",
+	Short: createChecksumHelper,
+	RunE: func(_ *cobra.Command, _ []string) error {
+		if err := assertDirExists(); err != nil {
+			return err
+		}
+		return check.CreateChecksum(dir)
+	},
+}
+
+const checkChecksumHelper = "Checks checksum of downloaded files."
+
+var checkChecksumCmd = &cobra.Command{
+	Use:   "check",
+	Short: checkChecksumHelper,
```

Comment:
A gente não precisa de uma constante para um texto curto

```suggestion
var checkChecksumCmd = &cobra.Command{
	Use:   "check",
	Short:  "Checks checksum of downloaded files.",
```